snippets/ram-console/ram-console.conf

```diff
@@ -0,0 +1,9 @@
+#
+# Copyright 2024 NXP
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Enable Console
+# CONFIG_UART_CONSOLE is not set
+CONFIG_RAM_CONSOLE=y
+CONFIG_RAM_CONSOLE_BUFFER_SECTION=y
```

Comment:
CONFIG_KERNEL_DIRECT_MAP=y as it is used in ram_console.c

Reply:
it is selected by the CONFIG_RAM_CONSOLE_BUFFER_SECTION. Thanks!